docs/organizations/copying-code-patterns-between-repositories.md

```diff
@@ -0,0 +1,34 @@
+# Copying code patterns between repositories
+
+Copy [tool and pattern configurations](../repositories-configure/code-patterns.md) in bulk between your repositories to help you bootstrap and standardize the coding standards in your organization.
+
+For example, when adding new repositories on Codacy you can copy the tool and pattern configurations from an existing repository that's already configured, and then tweak and adapt the settings for your new repositories.
+
+{%
+    include-markdown "../repositories-configure/code-patterns.md"
+    start="<!--code-patterns-copy-start-->"
+    end="<!--code-patterns-copy-end-->"
+%}
+
+To copy the tool and pattern configurations from one repository to other repositories:
+
+1.  Open your organization **Patterns** page.
+
+1.  Follow the instructions to select:
```

Comment:
```suggestion
2.  Follow the instructions to select:
```

Reply:
This is actually a [valid syntax for ordered lists](https://www.markdownguide.org/basic-syntax#ordered-lists), since the generated HTML list will have the correct list item numbers.

The advantage of always writing `1.` is that we don't have to remember to re-order the list items if we add or delete a step, or change the order of the steps in an ordered list. :nerd_face:

Reply:
wow :O